.buildkite/pipeline.yml

```diff
@@ -61,34 +62,20 @@ steps:
             allow_failure: true
           - step: "int-test"
             allow_failure: true
-
-      - label: "E2E Test"
-        key: "e2e-test"
-        command: ".buildkite/scripts/e2e_test.sh"
-        agents:
-          provider: "gcp"
-        depends_on:
-          - step: "unit-test"
-            allow_failure: false
-          - step: "int-test"
-            allow_failure: false
```

Comment:
This step must be recovered.
It was introduced in this PR https://github.com/elastic/fleet-server/pull/2646